components/service/pocket/src/test/java/mozilla/components/service/pocket/helpers/ext/HttpUrl.kt

```diff
@@ -2,6 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package mozilla.components.service.pocket
+package mozilla.components.service.pocket.helpers.ext
 
-class PlaceholderTest
+import android.net.Uri
+import mozilla.components.support.ktx.kotlin.toUri
+import okhttp3.HttpUrl
+
+fun HttpUrl.toUri(): Uri = toString().toUri()
```

Comment:
Is this helper used anywhere?

Reply:
No, I forgot to take it out. In fact, it didn't compile! 😲 I added this module's tests to my pre-push hook to prevent that in the future.